.github/workflows/jest.yml

```diff
@@ -0,0 +1,30 @@
+name: jest
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  jest:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
```

Comment:
I think we test against only 14 in Renovate

Reply:
```suggestion
        node-version: [14.x]
```